stdlib/Pkg/src/Pkg.jl

```diff
@@ -85,7 +85,10 @@ custom METADATA setup.
 init(meta::AbstractString=DEFAULT_META, branch::AbstractString=META_BRANCH) = Dir.init(meta,branch)
 
 function __init__()
+    Base.PKG_MODULE_REF[] = Pkg
```

Comment:
Gross, I hope this goes away eventually.

Reply:
Same here :)
cc @KristofferC